src/commonHelper.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as strings from './strings';
+import * as Constants from './constants';
+import { CommandObserver } from './CommandObserver';
+import * as compareVersions from 'compare-versions';
+const regex = /(?<="Microsoft.DataTools.Schema.Tasks.PostgreSql.Sdk\/)(.*?)(?=")/;
+
+export async function checkProjectVersion(minRequiredSDK: string, maxRequiredSDK: string, projects: string[], commandObserver: CommandObserver) : Promise<string[]> {
+	let unsupportedProjectsMap: string[] = [];
```

Comment:
There are places where spaces mixed with tabs, we need to converge on spaces, me thinks.

Reply:
Yes, we need to converge. But maybe on tabs. That is what they have been following.